aws.yaml

```diff
@@ -421,9 +421,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.3.2
+  name: v9.3.3
 spec:
   state: active
+  date: 2020-06-05T15:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.2
+    - name: chart-operator
+      version: 0.13.0
+    - name: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.4
+    - name: coredns
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.1
+    - name: nginx-ingress-controller
+      componentVersion: 0.30.0
+      version: 1.6.12
+    - name: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  components:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.7.1
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      version: 0.23.9
+    - name: kubernetes
+      version: 1.16.9
+    - name: containerlinux
+      version: 2345.3.1
+    - name: calico
+      version: 3.10.1
```

Comment:
Are we upgrading calico to 3.10.4? re: https://github.com/giantswarm/giantswarm/issues/11307#issue-629063741

Given what you said, "This proves it's not enough to change calico version metadata in release definition for that version to be actually used." cc @MartaD @MarcelMue

Reply:
Not in this release